delfin/drivers/ibm/storwize_svc/ssh_handler.py

```diff
@@ -0,0 +1,401 @@
+# Copyright 2020 The SODA Authors.
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+import time
+
+import paramiko
+import six
+from oslo_log import log as logging
+from oslo_utils import units
+
+from delfin import exception, utils
+from delfin.common import constants
+from delfin.drivers.utils.ssh_client import SSHPool
+
+LOG = logging.getLogger(__name__)
+
+
+class SSHHandler(object):
+    OID_ERR_ID = '1.3.6.1.4.1.2.6.190.4.3'
+    OID_SEQ_NUMBER = '1.3.6.1.4.1.2.6.190.4.9'
+    OID_LAST_TIME = '1.3.6.1.4.1.2.6.190.4.10'
+    OID_OBJ_TYPE = '1.3.6.1.4.1.2.6.190.4.11'
+    OID_OBJ_NAME = '1.3.6.1.4.1.2.6.190.4.17'
+    OID_SEVERITY = '1.3.6.1.6.3.1.1.4.1.0'
+
+    TRAP_SEVERITY_MAP = {
+        '1.3.6.1.4.1.2.6.190.1': constants.Severity.CRITICAL,
+        '1.3.6.1.4.1.2.6.190.2': constants.Severity.WARNING,
+        '1.3.6.1.4.1.2.6.190.3': constants.Severity.INFORMATIONAL,
+    }
+
+    SEVERITY_MAP = {"warning": "Warning",
+                    "informational": "Informational",
+                    "error": "Major"
+                    }
+
+    SECONDS_TO_MS = 1000
+
+    def __init__(self, **kwargs):
+        self.ssh_pool = SSHPool(**kwargs)
+
+    @staticmethod
```

Comment:
It seems that some function in this class has no relationship to this ssh class(such as `handle_split`, `parse_alert`, `change_capacity_to_bytes`, `parse_string` and so on), in my opinion it would be better to be put in another place.